core-web/src/main/java/org/silverpeas/core/webapi/search/ResultEntity.java

```diff
@@ -0,0 +1,140 @@
+package org.silverpeas.core.webapi.search;
+
+import org.silverpeas.core.index.search.model.SearchResult;
+import org.silverpeas.core.util.StringUtil;
+
+import javax.xml.bind.annotation.XmlElement;
+
+/**
+ * @author Nicolas Eysseric
+ */
+public class ResultEntity {
+
+  @XmlElement(defaultValue = "")
+  private String name;
+
+  @XmlElement(defaultValue = "")
+  private String description;
+
+  @XmlElement
+  private String creationDate;
+
+  @XmlElement
+  private String updateDate;
+
+  @XmlElement
+  private String type;
+
+  @XmlElement
+  private String id;
+
+  @XmlElement
+  private String componentId;
+
+  @XmlElement
+  private String thumbnailURL;
+
+  public static ResultEntity fromSearchResult(SearchResult gsr) {
+    return new ResultEntity(gsr);
+  }
+
+  private ResultEntity(SearchResult gsr) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)